### ISSUE TITLE
Training Attempt #1 

### ISSUE THREAD

**Issue by faseelmo** [faseelmo/noc_latency_predictor#22]
Train for 100 Epochs using MAE (L1) then use that weights to train on MSE (L2)

**Comment by faseelmo** [comment]
MAE and RMSE giving pretty much the same results.

**Comment by faseelmo** [comment]
Once MAE reaches its local minima try training using that weights and lowering the learning rate.

**Comment by faseelmo** [comment]
Model is overfitting if we train further than 200 Epoch
